data/tokens.json

```diff
@@ -3117,10 +3109,6 @@
           "value": "{Error.gtmds_color_error-3}",
           "type": "color"
         },
-        "gtmds_hint_color": {
```

Comment:
This one also...
Maybe we need a quick huddle as I cannot follow what you wanted to achieve :)